docs/code/snippets/custom_condition.py

```diff
@@ -12,7 +12,7 @@ def file_exists(file):
 @app.task(daily & file_exists("data.csv"))
 def do_things():
     "Task that runs once a day when data.csv exists"
-    ...
```

Comment:
I think these could be excluded (the three dots indicate just a placeholder here to show the functions should not be this short).